mmv1/third_party/terraform/services/apphub/data_source_apphub_discovered_service_test.go

```diff
@@ -0,0 +1,114 @@
+package apphub_test
+
+import (
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"github.com/hashicorp/terraform-provider-google/google/acctest"
+)
+
+func TestDataSourceApphubDiscoveredService_basic(t *testing.T) {
+	t.Parallel()
+
+	context := map[string]interface{}{
+		"random_suffix": acctest.RandString(t, 10),
+	}
+
+	acctest.VcrTest(t, resource.TestCase{
+		PreCheck:                 func() { acctest.AccTestPreCheck(t) },
+		ProtoV5ProviderFactories: acctest.ProtoV5ProviderFactories(t),
+		Steps: []resource.TestStep{
+			{
+				Config: testDataSourceApphubDiscoveredService_basic(context),
+			},
+		},
+	})
+}
+
+func testDataSourceApphubDiscoveredService_basic(context map[string]interface{}) string {
+	return acctest.Nprintf(
+		`
+resource "google_project" "service_project" {
+	project_id ="<%= ctx[:vars]['service_project_attachment_id'] %>-%{random_suffix}"
+	name = "Service Project"
+	org_id = "<%= ctx[:test_env_vars]['org_id'] %>"
+}
+
+resource "google_project_iam_binding" "apphub_service_project_iam_binding" {
```

Comment:
I'm not sure how this will work, but we may ultimately need this permission to be added at the org level in our 3 test environments, so that it is inherited by newly created projects.

Reply:
done